mediaire_toolbox/transaction_db/model.py

```diff
@@ -254,14 +282,18 @@ def read_dict(self, d: dict):
 
 class UserTransaction(Base):
 
-    """for multi-tenant pipelines, transactions might be associated with users"""
+    """for multi-tenant pipelines, transactions might be associated w/ users"""
     __tablename__ = 'users_transactions'
 
     user_id = Column(Integer, ForeignKey('users.id'),
                      primary_key=True)
     transaction_id = Column(Integer, ForeignKey('transactions.transaction_id'),
                             primary_key=True)
 
+    @validates()
```

Comment:
these lines are not doing anything, right? perhaps we can remove them if they are not used?

Reply:
Yes, but I thought this might encourage people to use tha validation if in the future DateTime fields are added. But yes, I can remove the unnecessary clutter.

Reply:
done